Aug 14 00:30:00 abc-MBP newsyslog[79597]: logfile turned over
Sun Aug 14 08:23:55.927 INFO  Ignoring non-dps events while not in monitoring period/ Ignoring DPS event while in monitoring period
Sun Aug 14 09:50:18.749 WARN  Could not set property APPLE80211_REGKEY_AWDL_MAC_ADDRESS_IN_USE to 1
Sun Aug 14 10:51:19.239 DEBUG [fcfff715-82cc-400d-88b4-56f25b5a5239:networkName:BSSID:]
Sun Aug 14 10:51:19.251 ERROR Usb Host Notification Error Apple80211Set: Device power is off seqNum 16 Total 4 chg 1 en0
Sun Aug 14 10:51:19.272 INFO  _handleLinkEvent: WiFi is not powered. Resetting state variables.
Sun Aug 14 10:51:19.316 DEBUG Usb Host Notification Error Apple80211Set: Device power is off seqNum 17 Total 4 chg 1 en0
Sun Aug 14 10:51:22.602 DEBUG sharingd (542) is not entitled for com.apple.wifi.join_history, will not allow request

Jan 02 21:14:54 innodellix (udev-worker)[1]: Started Accounts Service.
Jan 02 21:14:54 innodellix systemd[1]: Started Rule-based Manager for Device Events and Files.
Jan 02 21:14:54 innodellix systemd[1]: Stopped libvirtd-config.service: Deactivated successfully.
Jan 02 21:14:54 innodellix systemd[1]: Finished Libvirt Virtual Machine Management Daemon - configuration.
Jan 02 21:14:54 innodellix systemd[1]: suid-sgid-wrappers.service: Deactivated successfully.
Jan 02 21:14:54 innodellix systemd[1]: Finished Create SUID/SGID Wrappers.
Jan 02 21:14:54 innodellix systemd[1]: Started Cleanup of Snapper Snapshots.

# Date and Time
07:46:34                       Data backup created
10:51:19.251                   Event logged
2023/10/02 11:47:39:850        Task completed
2022-09-09 11:48:34,534        Last login attempt failed
2022-09-22T07:46:34.171800155Z Event logged for debugging purposes

# IP
Suspicious activity detected from IP:  10.0.0.123
Connected to server at IP address:     192.168.0.1
No connection with remote host:        172.16.0.254
Request received from client IP:       2001:db8:0:0:0:ff00:42:8329
Multiple failed login attempts:        2001:db8::ff00:42:8329

# Ip v 6
Log 2001:0db8:85a3:0000:0000:8a2e:0370:7334 asdf
Log 2001:db8:85a3::8a2e:370:7334 asdf
Log 2001:db8:85a3::8a2e:192.0.2.33 asdf
Log fe80::7a31:c1ff:fe02:bc33 asdf
Log [2001:db8:85a3::8a2e:370:7334]:8080 asdf
Log 2001:db8::1234:5678 asdf
Log fe80::1ff:fe23:4567:890a%eth0 asdf

# Pointers
Error occurred at memory address 0x8c2a0aeb
Received data from 0x25f4a93b, processing now
Kernel module loaded at address 0xd7b3b2f446e2c21b
Failed to read data from address 0x96dc1ac2a6db0b40
Copying data from 0x7f6a2c21 to 0x706a2c21

# Numbers
Duration: 42 seconds
High CPU usage detected: 95%
Value must be between 1 and 10, received: 15
Records created: 1000
Calculation result: 3.14159

# URL
http://example.com/path/to/resource?param1=value1&param2=value2
http://api.example.com/v2/resources?search=keyword&page=1
https://api.example.org/api/v1/users?name=JohnDoe
https://www.openai.com/docs/api?apikey=abc123
https://api.example.org/v1/users/123/edit?param=true&flag=false

# HTTP Methods
User with IP 192.168.1.105 initiated a GET request for /about/us
PROPFIND /path/to/ressource/ HTTP/1.1" 207 4471 "-" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Mobile Safari/537.36"
A POST request by 192.168.1.106 to /api/create-account failed
Received a PUT operation on /api/v1/users/update-profile
During routine checks at 14:13:40, a PATCH request was detected
Unauthorized DELETE attempt on /api/v2/data/secure by user

# UUID
User logged in, user ID:     5f7d1bce-81ab-4a87-af78-9a37f26c58b1
Invalid request:             9f5f25be-6b08-4aeb-a10a-023b86581b1f
Database connection failed:  3e169d8f-c971-4e6b-8ff3-90ac2c912e6c
Task completed successfully: d8f40210-7ef4-4d2a-b482-25e8b3a9eae8
Processing request:          2a54e369-12f7-4ec2-9dc1-685e05b9017a

# Unix file paths
Failed to open directory:   /usr/local/bin/
File successfully created:  /var/log/application.log
Reading configuration file: /etc/nginx/nginx.conf
File not found:             ~/project/user/documents/report.pdf
Processing file:            ~/discovery/alfa/project/src/main.js

# Keywords
[INFO ] Task completed. Status: true (result: null)
[WARN ] Invalid input received (isValid: false)
[ERROR] Connection failed: null (isConnected: false)
[TRACE] Configuration loaded: true (version: N/A, env: null)
[DEBUG] Invalid request: {active: true, age: null}

# Quotes
Request made to sync data with our partner "Acme Corp's" servers.
Cache miss detected, fetching fresh copy from "Main Server".
User "Alice" logged out. All associated sessions cleared.
Purchase for item "Red Bicycle" was successful. Thank you!
Configuration file "PrimaryConfig" loaded. Starting operations.
Processing new order for customer "JohnDoe". Stand by.

# Key-value pairs
ts=08:11:36 caller=module_service.go:59 module=table-manager
ts=08:11:36 caller=module_service.go:59 module=ingester-querier
ts=08:11:36 caller=lifecycler.go:530 msg="file path is empty"
ts=2022-09-22 caller=ring.go:275 msg="ring doesn't exist"
ts=2022-09-21 caller=client.go:247 msg="value is nil"

# Processes
kernel[0]: Page fault at address 0xabcdef
mysqld[789]: Connected from 127.0.0.1
docker[654]: Container abcdefgh started successfully
ssh-d[222]: Accepted publickey for root from 192.168.1.20
cron[101]: Running scheduled job ID 23456


